inductiva/resources/machines_base.py

```diff
@@ -250,6 +250,7 @@ def _register_machine_group(self, **kwargs):
             body.get("idle_seconds"))
         self._auto_terminate_ts = self._iso_to_datetime(
             body.get("auto_terminate_ts"))
+        self.ram_gb = body.get("ram_gb")
```

Comment:
Is this total ram or ram per core ?

Reply:
Total ram

Reply:
Then i would recommend changing it to self.total_ram_gb. Since gcp has ram per vCpus, we need to make a distinction.